client/i18n/locales/english/certs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "certNames": {
+    "Responsive Web Design": "RWD Hacked by nhcarrigan"
```

Comment:
Why not use the dashed-name here as a key?

Reply:
The dashed name isn't always available depending on how we fetch the data.

Reply:
I would prefer if we forced it's existence, in most of these cases.

We can hard-code for this PR, but then will annoy ourselves when we make another change for SEO or similar.

Reply:
I'll poke at that.

Reply:
I cheated and used `dasherize` where the dashedName wasn't readily available. :)

Reply:
That might be fine, but, a short while ago, we did spend time specifically removing the use of `dasherize`.